API-6450 Archives highlights (#681)
livechat/livechat-public-docs: content/messaging/agent-chat-api/v3.2/index.mdx

| `filters.surveys.<survey>`                            | No       | `array`   | **\*\*** **described below**                                                                                                 |
| `filters.events.types`                                | No       | `array`   | Array of [Event](#events) types. Duplicates are ignored.                                                                     |
| `pagination`                                          | No       | `object`  |                                                                                                                              |
| `pagination.page`                                     | No       | `number`  | Default: 1, min: 1, max: 1000                                                                                                |
| `pagination.limit`                                    | No       | `number`  | Default: 25, min: 0, max: 100                                                                                                |

> | `highlights`                                          | No       | `object`  | To enable highlights with default parameters, just pass an empty object. Should be used in conjunction with `filters.query`. |

Reviewer: ```suggestion
| `highlights`                                          | No       | `object`  | Use it to highlight the match of `filters.query`. To enable highlights with default parameters, pass an empty object. |
```

> | `highlights.pre_tag`                                  | No       | `string`  | HTML tags to use for the highlighted text, default value: `<em>`. Should be used in conjunction with `highlights.post_tag`.  |

Reviewer: ```suggestion
| `highlights.pre_tag`                                  | No       | `string`  | An HTML tag to use for highlighting the matched text; default: `<em>`. Use it together with `highlights.post_tag`.  |
```

> | `highlights.post_tag`                                 | No       | `string`  | HTML tags to use for the highlighted text, default value: `</em>`. Should be used in conjunction with `highlights.pre_tag`.  |

Reviewer: ```suggestion
| `highlights.post_tag`                                 | No       | `string`  | An HTML tag to use for highlighting the matched text; default: `</em>`. Use it together with `highlights.pre_tag`.  |
```
